README.md

```diff
@@ -48,3 +48,44 @@ by looking for at least one `coding` from the following known COPD Condition cod
   - "code": "J44.9"
 - "system": "http://snomed.info/sct"
   - "code": "404684003"
+
+## How To Run
+
```

Comment:
Nitpick: shouldn't need a space after the title (technically doesn't do anything)
```suggestion
```